src/bandersnatch/configuration.py

```diff
@@ -48,7 +48,9 @@ def __init__(self, config_file: Optional[str] = None) -> None:
             Path to the configuration file to use
         """
         self.found_deprecations: List[str] = []
-        with importlib.resources.path("bandersnatch", "default.conf") as config_path:
+        with importlib.resources.path(  # type: ignore
+            "bandersnatch", "default.conf"
+        ) as config_path:
```

Comment:
@SanketDG - Any ideas why this error occurs? I haven't dug into it ... Maybe because in 3.6 we use the pulled in dependency?

I might move 4.1 to be >= 3.7.0 since Docker containers make moving to newer python so much easier these days ...

Reply:
Yup, the library does not seem to have type hints
https://gitlab.com/python-devs/importlib_metadata/-/issues/10